src/main/java/com/github/jvdberg08/commandhandler/argument/FixedCommandArgument.java

```
@@ -0,0 +1,23 @@
+package com.github.jvdberg08.commandhandler.argument;
+
+import org.apache.commons.lang.ArrayUtils;
+
+public class FixedCommandArgument implements CommandArgument<String> {
```

Could you delete StringCommandArgument.java and rename this to StringCommandArgument?

---

Would that be a good idea? Right now StringCommandArgument allows an easy to use form for one specific string (e.g. teleport in your wiki). Right now FixedCommandArgument always requires and array which could be annoying when using it for just one string. An alternative would be to change the constructor to (String... fixed) parameters, but that would also allow no paramenter which isn't good either.

---

I don't think being able to pass an empty array should be an issue, because who would even do that?: :stuck_out_tongue_winking_eye:
Maybe keeping it as a String[] instead of String... will give people the idea that something should really be passed. Alternatively you can throw an exception if an empty string is passed. I don't think should warrant a whole new class, since the class that has the String[] parameter will have this issue anyways.